examples/golang/README.md

```diff
@@ -0,0 +1,32 @@
+# Upload Object to Ceph Cluster using AWS sdk
+
+This golang example shows us how to use a ceph cluster object gateway in place of an aws s3. A ceph cluster gateway is directly compatible with aws sdk or cli. So, you don't need to worry about rewriting your codebase. In more examples to come, you would be shown how to create and get notifications on a particular bucket and every other thing you can do with your aws cli or sdk
```

Comment:
IMO, you can skip this intro. users that look in the examples would already know about that

Reply:
OK, would take this out of the documentation.